hyperjumptech/monika
# added: stub getPublicNetworkInfo to make test run faster

# Monika Pull Request (PR)  
fix #400 

## What feature/issue does this PR add  
1. add stub for getPublicNetworkInfo

## How did you implement / how did you fix it  
1.  add stub for getPublicNetworkInfo

## How to test  
1. run `npm run test`
2. see no red elapsed time and no logging of network info
<img width="893" alt="Screen Shot 2021-09-10 at 17 19 36" src="https://user-images.githubusercontent.com/5974862/132840050-f418c99b-2095-45a0-b8f6-81a6869b1924.png">
